content/press.adoc

```diff
@@ -90,8 +90,8 @@ volunteers so availability and time may be limited.
 
 === US press contacts
 
-* link:https://github.com/rtyler[R. Tyler Croy] (_english_) - Board member - `rtyler[at]brokenco.de`
-* link:https://github.com/kohsuke[Kohsuke Kawaguchi] (_english, japanese_) - Project founder / board member - `kk[at]kohsuke.org`
+* link:https://github.com/halkeye[Gavin Mogan] (_english_) - Board member - `jenkins[at]gavinmogan.com`
+* link:https://github.com/kohskue[Kohsuke Kawaguchi] (_english, japanese_) - Project founder / board member - `kk[at]kohsuke.org`
```

Comment:
```suggestion
* link:https://github.com/kohsuke[Kohsuke Kawaguchi] (_english, japanese_) - Project founder / board member - `kk[at]kohsuke.org`
```

Reply:
Because https://github.com/kohskue is HTTP/404 😅

Reply:
Are the other contacts (EU, CIS, China) still valid and should stay there?